docs/02-Scales.md

```diff
@@ -231,7 +231,7 @@ minUnit | String | 'millisecond' | The minimum display format to be used for a t
 
 When providing data for the time scale, Chart.js supports all of the formats that Moment.js accepts. See [Moment.js docs](http://momentjs.com/docs/#/parsing/) for details.
 
-#### Display Formats
+### Display Formats
```

Comment:
Ah, I see that now. So, would you advise to just remove the anchor links for now? Another option is to insert invisible anchor tags. Third option is to leave it alone, but as a user the broken link confused me. What do you think?